frontend/angular.json

```diff
@@ -29,7 +29,8 @@
               "./node_modules/prismjs/themes/prism-okaidia.css",
               "./node_modules/primeng/resources/themes/saga-blue/theme.css",
               "./node_modules/primeng/resources/primeng.min.css",
-              "./node_modules/primeicons/primeicons.css"
+              "./node_modules/primeicons/primeicons.css",
+              "./node_modules/primeflex/primeflex.css"
```

Comment:
[Primeflex](https://github.com/primefaces/primeflex)

So we can use 

[flexbox](https://www.primefaces.org/primeng/showcase/#/primeflex/flexbox)
and 
[formlayout](https://www.primefaces.org/primeng/showcase/#/primeflex/formlayout)